chall/str_replace.cpp

```diff
@@ -0,0 +1,28 @@
+#include <iostream>
+#include <string>
+using namespace std;
+
+string ReplaceString(string subject, const string& search, const string& replace) {
```

Comment:
what about snake_case? The C++ book we are following recommends it I think.

Reply:
I just copy and paste this function. Fixed.